### DISCUSSION TITLE
What is backed up?

### DISCUSSION THREAD

**Discussion by BenPortner** [seedvault-app/seedvault#198] [Q&A]
Dear Seed Vault developers,

first off, thanks for the great app. A proper, open source backup utility for AOSP is long overdue!

Maybe I just overlooked it but I couldn't find it: What exactly is backed up by Seed Vault? Is it the complete `data`-partition? Does it include `data/media` (unlike a TWRP backup)? Does it include other partitions like `system`, `cache`, `vendor`? I'm curious to know :)

Thanks and all the best!
Ben

**Comment by grote** [comment]
It uses Android's backup API, so it backs up data from apps, if they provide any data for backup.

https://developer.android.com/guide/topics/data/backup

**Reply by radio-flow** [reply]
what about system settings etc.?
I've always wondered where they are stored

Thank you!

**Reply by grote** [reply]
Most of those should get backed up by the system Settings app.
